tasks/set-batch-block-limit.ts

```diff
@@ -11,5 +13,7 @@ task("set-batch-block-limit", "Sets a new batch block limit")
     const safeContractFactory = await hre.ethers.getContractFactory("ERC20Safe");
     const safe = safeContractFactory.attach(safeAddress).connect(adminWallet);
 
-    await safe.setBatchBlockLimit(taskArgs.blocks);
+    await safe.setBatchBlockLimit(taskArgs.blocks, getDeployOptions(taskArgs));
```

Comment:
Log line is missing (present in other snippets).

Reply:
log lines are kinda used only for debugging - printing the whole result here does not help to much, it's to verbose. Maybe some status printing for all actions, in another PR